src/packets.c

```diff
@@ -1223,30 +1223,38 @@ TbBool process_dungeon_control_packet_clicks(long plyr_idx)
             slb = get_slabmap_block(subtile_slab_fast(stl_x), subtile_slab_fast(stl_y));
             if (slb->room_index)
                 {
-                    struct Room* room = room_get(slb->room_index);
+                    room = room_get(slb->room_index);
                     destroy_room_leaving_unclaimed_ground(room);
                 }
             unset_packet_control(pckt, PCtr_LBtnRelease);
         }
         break;
     case PSt_KillCreatr:
+            thing = get_creature_near(x, y);
+            if (!thing_is_creature(thing))
```

Comment:
The indentation seems wrong here, the 'if' is indented more than the 'else' statement.